app/app_darwin.go

```diff
@@ -33,3 +36,20 @@ func (app *fyneApp) OpenURL(url *url.URL) error {
 	cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
 	return cmd.Run()
 }
+
+func (app *fyneApp) SendNotification(notify *fyne.Notification) {
+	title := escapeNotifyString(notify.Title)
+	content := escapeNotifyString(notify.Content)
+	template := `display notification "%s" with title "%s"`
+	script := fmt.Sprintf(template, content, title)
+
+	err := exec.Command("osascript", "-e", script).Start()
```

Comment:
You can only access UNUserNotificationCenter if you are running a correctly bundled app with appropriate entitlements signed into the application.

I don't think that is a reasonable requirement for fyne app developers and means you can't run them using "go run" or from "go install"ed binaries.

Reply:
I could probably do some checks to verify whether or not this route will succeed - however it means that developers can no longer compile below 10.14... Do we want to do that?

Reply:
Update on this: Using UNUserNotificationCenter without a proper app bundle crashes hard with a failed assertion that I cannot find how to check for. It's deep in undocumented apple code...

Reply:
Okay, so let it be this way for the first iteration. But you should add an issue for “support OSX notifications from the app instead of Mac script”. You can assign it to me, if you want :).